js/mexc.js

```diff
@@ -39,6 +40,11 @@ module.exports = class mexc extends Exchange {
                 'fetchDeposits': true,
                 'fetchFundingRate': true,
                 'fetchFundingRateHistory': true,
+                'fetchFundingRates': false,
+                'fetchIndexOHLCV': false,
+                'fetchIsolatedPositions': false,
```

Comment:
We can emulate this, from fetchPositions select the ones that are isolated

Reply:
I will work on this